src/observerBatching.ts

```diff
@@ -1,25 +1,17 @@
 import { configure } from "mobx"
 
-import { getGlobal, getSymbol } from "./utils"
-
-interface IBatchedUpdates {
-    batchedUpdates<A, B>(callback: (a: A, b: B) => any, a: A, b: B): void
-    batchedUpdates<A>(callback: (a: A) => any, a: A): void
-    batchedUpdates(callback: () => any): void
+export function defaulNoopBatch(callback: () => void) {
+    callback()
 }
 
-const observerBatchingConfiguredSymbol = getSymbol("observerBatching")
-
-export const observerBatching = (reactionScheduler?: IBatchedUpdates) => {
```

Comment:
I believe this was mentioned in readme, some people might have been using it so better to alias it to `setBatch`.

And readme should be updated as well.